SYCL/GroupAlgorithm/exclusive_scan_over_group.cpp

```diff
@@ -1,6 +1,7 @@
 // UNSUPPORTED: cuda || hip
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
+// UNSUPPORTED: ze_debug4
```

Comment:
```suggestion
// UNSUPPORTED: ze_debug4,ze_debug-1
```

Reply:
ZE_DEBUG=-1 includes ZE_DEBUG=4.
So, please mark both ze_debug-1 and ze_debug4.

Reply:
Argh, I misread it originally - thought that it was a positive "one" with different (for some reason) spelling. Now it makes sense.

Thanks!